media/css/pocket/components/add.scss

```diff
@@ -213,44 +212,54 @@
     display: flex;
     flex-direction: column;
 
+    h3 + * {
```

Comment:
NIt: potentially a broad selector, could this just be `h3 + p`?

Reply:
Actually looking at the DOM more closely, this is probably fine 👍

Reply:
lol, just made the change. It's probably only ever going to be a `p` following the heading in any case.